.cirrus.yml

```diff
@@ -11,22 +11,22 @@ cargo_task_tempate: &cargo_task_template
       - cat Cargo.lock
   setup_script:
     - apt-get update
-    - apt-get -y install llvm clang libelf-dev libpcap-dev python3-pip
+    - apt-get -y install llvm clang libelf-dev libpcap-dev python3-pip make jq
     - rustup component add rustfmt
     - rustup component add clippy
   before_cache_script: rm -rf $CARGO_HOME/registry/index
 
 unittest_task:
   << : *cargo_task_template
-  build_script: cargo build --verbose
+  build_script: CARGO_BUILD_OPTS=-v make V=1
   test_script: cargo test --verbose
   check_script:
     - cargo fmt --check
     - cargo clippy -- -D warnings
 
 benchmark_task:
   << : *cargo_task_template
-  build_script: cargo build -F benchmark --verbose
+  build_script: make ebpf V=1 && cargo build -F benchmark --verbose
```

Comment:
I think we should support every common target in the Makefile directly, at least `test` and probably `benchmark`. Otherwise the `build` vs other cargo targets makes things inconsistent and let the user think using cargo directly is supported; whereas it's now a low-level tool.

Same applies for `cargo build --release`, which should have instead a `release` target in the Makefile.

Having said that, it's very nice to have an `ebpf` target and be able to use cargo, but as low level tools.

Reply:
yes, that's my idea as well (for the final result)

Reply:
about cargo release, I'm not sure we need a separate target.
I think we can simply do something like:

`make CARGO_BUILD_OPTS=--release V=1 -j$(nproc)`
instead of:
`make release V=1 -j$(nproc)`

WDYT?

Reply:
> Just for testing, I verified that even more complex commands still work (although can be run separately): `CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER=sudo RUST_BACKTRACE=1 make test CARGO_CMD_OPTS="--features=test_cap_bpf,debug core::probe::kernel::raw_tracepoint::tests::init_and_attach -- --nocapture --show-output"`

Nice!

> about cargo release, I'm not sure we need a separate target. I think we can simply do something like:
> 
> `make CARGO_BUILD_OPTS=--release V=1 -j$(nproc)` (in the latest iteration should be `s/CARGO_BUILD_OPTS/CARGO_CMD_OPTS/`) instead of: `make release V=1 -j$(nproc)`

While I agree `--release` is just a flag we can give to Cargo, it's really a target we use a lot and IMO it's easier and more user friendly to maintain it as a target.

Reply:
release target is now part of the series